Sources/EmojiText/EmojiText.swift

```diff
@@ -323,6 +324,18 @@ public struct EmojiText: View {
 
 // swiftlint:disable force_unwrapping
 struct EmojiText_Previews: PreviewProvider {
+
+    struct EmojiTextWithSlider: View {
+        @State private var emojiSize: CGFloat = 20
+
+        var body: some View {
+            EmojiText(verbatim: "Hello World :mastodon: with a remote emoji",
+                      emojis: emojis)
+            .emojiSize(emojiSize)
+            Slider(value: $emojiSize, in: 1...50)
+        }
+    }
```

Comment:
Wrapped because the binding value passed to the Slider does not change when using Preview.